src/Silakov.Homework6/Managers/AtmManager.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Silakov.Homework6.Interfaces;
+using Silakov.Homework6.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Silakov.Homework6.Managers
+{
+    class AtmManager : Atm, IAtmManager
```

Comment:
Please think about the real world example: can a manager of ATMs be just a more specific ATM itself